saleor/core/models.py

```diff
@@ -0,0 +1,14 @@
+from django.conf import settings
+from django.db import models
+
+
+class BaseNote(models.Model):
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL, blank=True, null=True,
+        on_delete=models.SET_NULL)
+    date = models.DateTimeField(db_index=True, auto_now_add=True)
+    content = models.TextField()
+    is_public = models.BooleanField(default=True)
+
+    class Meta:
```

Comment:
This model has to be abstract. We don't want to save instances of `BaseNote` in the database, but only the derived models such as `OrderNote` or `CustomerNote`.